mindone/models/dit.py

```diff
@@ -308,11 +375,30 @@ def construct(self, x, mask=None):
             v = v.view(v_b, v_n, h, -1).transpose(0, 2, 1, 3)
             if mask is None:
                 mask = ops.zeros((q_b, q_n, q_n), self.fa_mask_dtype)
-
-            out = self.flash_attention(
-                q.to(ms.float16), k.to(ms.float16), v.to(ms.float16), mask.to(self.fa_mask_dtype)
-            )
-
+            # FIXME: a trick to pad head dimensions from 72 to 128
```

Comment:
same, looks ugly, hope to provide a wrapper

Reply:
I wrote a wrapper. More suggestions are welcomed.

Reply:
nice work!